### ISSUE TITLE
2.0.x DB\Adapter escapeIdentifier()

### ISSUE THREAD

**Issue by max91** [phalcon/cphalcon#10370]
https://github.com/phalcon/cphalcon/blob/2.0.x/phalcon/db/adapter/pdo.zep#L380

Please fix this piece of pain. Because if to escape the names, your function does sql request invalid

**Comment by rvanderfeer** [comment]
Not sure if this is related, but I'm experiencing an issue where a table name which is a reserved word is not escaped.
In my case the Model is 'Group', and the table name 'group'.
Previous Phalcon version would escape group from mysql by adding backticks ( ``group`` ).
This no longer seems to be the case as I'm no seeing syntax errors such as:

```SQL
SQLSTATE[42000]: Syntax error or access violation: 1064
You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near
'group WHERE `group`.`id` = '1' LIMIT 1' at line 1
```

Note that the `group` occurrence in the WHERE clause is properly escaped, where the `group` in the SELECT part is not.

**Comment by andresgutierrez** [comment]
@rvanderfeer That is fixed in 2.0.2

**Comment by rvanderfeer** [comment]
@andresgutierrez Thank you for your quick reply.
I don't see a 2.0.2 tag / branch in github - am I correct to assume these changes are in master branch? 
Do you have an idea or indication of a release date?

**Comment by andresgutierrez** [comment]
@rvanderfeer It's not released yet, you can compile from the 2.0.x branch to obtain this version

**Comment by Green-Cat** [comment]
@andresgutierrez Fixed in 2.0.2
